clients/cli/src/main/java/org/apache/gravitino/cli/commands/Command.java

```diff
@@ -99,6 +100,16 @@ public static void setAuthenticationMode(String authentication, String userName)
 
   /** All commands have a handle method to handle and run the required command. */
   public abstract void handle();
+
+  /**
+   * verify the arguments. All commands have a verify method to verify the arguments.
+   *
+   * @return Returns itself via argument validation, otherwise exits.
+   */
+  public Command verify() {
```

Comment:
I'd call this `validate` rather than `verify`.
`verify`, to me, is an action to confirm that something happened.

Reply:
I agree validate is probably a better name.